libs/payments/cart/src/lib/checkout.service.ts

```diff
@@ -165,9 +165,23 @@ export class CheckoutService {
   }
 
   async postPaySteps(cart: ResultCart, subscription: StripeSubscription) {
-    // TODO: Add tax ID to customer
+    const { customer, currency } = subscription;
```

Comment:
Just thinking out loud here, instead of naming the variable `customer`, would a more accurate name of `customerId` make it more clear what this variable is and make it easier to read and understand?

From a typing perspective it's clear that it's the customer ID, since the type can only be `string`, but when reading the code it's not so clear whether this is a customer object or just a customer ID.